### ISSUE TITLE
Incorrect frame size due to box-sizing override on some sites

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/dict-lookup-chrome-ext#33]
```
What steps will reproduce the problem?
1.Just use it as usually

What is the expected output? What do you see instead?
- The frame should have rounded corners below, but it doesnt.

What version of the product are you using? On what operating system?
- Latest Chrome on Windows

Please provide any additional information below.
- Please see attached screen shot
```

Original issue reported on code.google.com by `haider....@gmail.com` on 21 Jan 2015 at 4:01

Attachments:
- [Dictionary Lookup Bug.jpg](https://storage.googleapis.com/google-code-attachments/dict-lookup-chrome-ext/issue-33/comment-0/Dictionary Lookup Bug.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.3.4.
```

Original comment by `max99x` on 21 Jan 2015 at 4:29
- Changed title: **Incorrect frame size due to box-sizing override on some sites**
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Looks really good now, thanks!
```

Original comment by `haider....@gmail.com` on 23 Jan 2015 at 11:53
